static/js/popovers.js

```diff
@@ -519,10 +521,14 @@ exports.register_click_handlers = function () {
         if (emoji_map_is_open) {
             // If the popover is already shown, clicking again should toggle it.
             popovers.hide_emoji_map_popover();
-            return;
+        } else { // popovers.hide_all();
```

Comment:
I don't think that it is an unwanted call. Purpose of this call is to hide other popovers like message actions popover, if they are open before opening the emoji picker popover. I think you should add it back.